modules/storages/app/common/storages/peripherals/storage_interaction/nextcloud_storage_query.rb

```diff
@@ -96,18 +96,19 @@ def storage_files(response)
     end
 
     def storage_file(file_element)
-      name = name(file_element)
+      location = name(file_element)
+      name = CGI.unescape(location.split('/').last)
 
       ::Storages::StorageFile.new(
         id(file_element),
-        CGI.unescape(name.split('/').last),
+        name,
         size(file_element),
         mime_type(file_element),
         nil,
         last_modified_at(file_element),
         created_by(file_element),
         nil,
-        "/#{name}"
```

Comment:
Did we missed this last time?

Reply:
yes, this is needed due to the removal of the trailing `/` of the base url